src/test/java/se/diabol/jenkins/pipeline/PipelineVersionContributorTest.java

```diff
@@ -150,18 +145,15 @@ public void testVersionContributorConfiguredManualTrigger() throws Exception {
         jenkins.waitUntilNoActivity();
 
         assertNotNull(secondProject.getLastBuild());
-
     }
 
     @Test
     public void testIsApplicable() throws Exception {
         PipelineVersionContributor.DescriptorImpl d = new PipelineVersionContributor.DescriptorImpl();
         assertTrue(d.isApplicable(jenkins.createFreeStyleProject("a")));
-
     }
 
-    @Test
-    @Bug(21070)
+    @Test // JENKINS-21070)
```

Comment:
Sure, it can be used for reports (even though we don't use Spock atm) but we can change back to using @Issue instead since it seems to be of interest.